app/assets/javascripts/petition-widget-builder.js

```diff
@@ -0,0 +1,103 @@
+(function($) {
+
+  // This is not good, but I would not like to make a request to fetch this HTML
```

Comment:
Why, I use it all the time when creating templates.

Then in js code I can just:

```
JST["my-template"]({someVar: "a"})
```